src/SFA.DAS.Tools.Support.Web/Views/Approvals/StopApprenticeship.cshtml

```diff
@@ -4,61 +4,90 @@
     @using SFA.DAS.Tools.Support.Web.Configuration
 }
 
-<div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-xl">
-            Stop an Apprenticeship
-        </h1>
-        <div asp-validation-summary="ModelOnly" class="govuk-error-message"></div>
-        <form asp-route="@ApprovalsRouteNames.StopApprenticeship" method="post">
-            <div class="govuk-form-group">
-
-                <label asp-for="ApprenticeshipId" class="govuk-label">
-                    Apprenticeship Id
-                </label>
-                <span id="ApprenticeshipId" class="govuk-error-message">
-                    @Html.ValidationMessageFor(m => m.ApprenticeshipId)
-                </span>
-                <span id="apprenticeship-id-hint" class="govuk-hint">
-                    The Id of the apprenticeship to stop.
-                </span>
-                <input asp-for="ApprenticeshipId" id="apprenticeshipId" class="govuk-input govuk-input--width-20" />
-            </div>
-            <div class="govuk-form-group">
-                <label asp-for="EmployerAccountId" class="govuk-label">
-                    EmployerAccount Id
-                </label>
-                <span id="EmployerAccountId" class="govuk-error-message">
-                    @Html.ValidationMessageFor(m => m.EmployerAccountId)
-                </span>
+@if (Model != null && Model.ApprenticeshipNotFound)
```

Comment:
As per our earlier conversation we should change this page when we do the bulk change task to allow bulk updates

Reply:
Will add this to the bulk updates task 👍